src/reference-manual/expressions.qmd

```diff
@@ -782,8 +782,10 @@ In addition to single integer indexes, as described in
 [the language indexing section](#language-indexing.section), Stan supports multiple indexing.
 Multiple indexes can be integer arrays of indexes, lower
 bounds, upper bounds, lower and upper bounds, or simply shorthand for
-all of the indexes.  A complete table of index types is given in the
-following table.
+all of the indexes.  If the upper bound is smaller than the lower bound, 
+the range is empty (unlike, e.g. in R). Upper bound and lower bound can be 
```

Comment:
"e.g." always gets a comma after it.  

Upper bound and lower bound -> The upper bound and lower bound

A complete table -> A complete list (rather than using "table" twice)